### PR TITLE
fix(PolicySystemsTab): RHICOMPL-1659 add the OS version column

### PR DESCRIPTION
**Pull request by skateman**
## Secure Coding Practices Checklist GitHub Link
- https://github.com/RedHatInsights/secure-coding-checklist

## Secure Coding Checklist
- [x] Input Validation
- [x] Output Encoding
- [x] Authentication and Password Management
- [x] Session Management
- [x] Access Control
- [x] Cryptographic Practices
- [x] Error Handling and Logging
- [x] Data Protection
- [x] Communication Security
- [x] System Configuration
- [x] Database Security
- [x] File Management
- [x] Memory Management
- [x] General Coding Practices

### CODE REVIEW

File: src/SmartComponents/PolicyDetails/PolicySystemsTab.js [hunk 1]
```diff
@@ -42,7 +42,11 @@ const PolicySystemsTab = ({ policy, systemTableProps }) => {
                         supported={ realProfile.supported }
                         ssgVersion={ realProfile?.ssg_version || realProfile?.ssgVersion } />;
                 }
-            }] : []]}
+            }] : [], {
+                key: 'osMinorVersion',
+                title: 'OS version',
```
Comment (vkrizan) [inline src/SmartComponents/PolicyDetails/PolicySystemsTab.js:47]: The other views display "Operating system" as title and "RHEL X.Y" within a cell. Would you mind making it consistent?

Comment (skateman) [reply src/SmartComponents/PolicyDetails/PolicySystemsTab.js:47]: Right, missed that...fixing

Comment (vkrizan) [reply src/SmartComponents/PolicyDetails/PolicySystemsTab.js:47]: There is still `OS version` instead of `Operating system`. :wink:

**Review by vkrizan — APPROVED** [review]
:+1:

**Pull request merged**